back/app/serializers/web_api/v1/user_serializer.rb

```diff
@@ -14,7 +14,7 @@ class WebApi::V1::UserSerializer < WebApi::V1::BaseSerializer
     :block_reason,
     :followings_count,
     :onboarding,
-    :last_acted_at,
+    :last_active_at,
```

Comment:
I think we should consider this a private attribute, and protect it just as we do with e.g. email on line 41. Non-privileged users shouldn't be able to get access to data that says when someone visited.